acdc/ioi/utils.py

```diff
@@ -198,9 +198,10 @@ def get_ioi_true_edges(model):
     for layer_idx in range(12):
         for head_idx in range(12):
             if (layer_idx, head_idx) not in all_nodes:
-                nodes_to_mask.append(
-                    TLACDCInterpNode(name=f"blocks.{layer_idx}.attn.hook_result", index = TorchIndex([None, None, head_idx]), incoming_edge_type=EdgeType.DIRECT_COMPUTATION),
-                )
+                for letter in ["q", "k", "v"]:
+                    nodes_to_mask.append(
+                        TLACDCInterpNode(name=f"blocks.{layer_idx}.attn.hook_{letter}", index = TorchIndex([None, None, head_idx]), incoming_edge_type=EdgeType.DIRECT_COMPUTATION),
```

Comment:
I think you want to remove `hook_{letter}_input`, `hook_{letter}`, and `hook_result` here